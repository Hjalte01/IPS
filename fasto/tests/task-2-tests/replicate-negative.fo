fun int main() =
    let n = replicate(~1, 1) in
    0
