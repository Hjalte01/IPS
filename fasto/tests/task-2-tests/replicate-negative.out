Interpreter error: Argument of "replicate" is negative: -1 at line 2, column 5
